educhain/engines/qna_engine.py

```
@@ -434,44 +613,34 @@ def generate_mcq_math(
             available_languages = [transcript.language_code for transcript in transcript_list]
-            
+
             if not available_languages:
                 raise ValueError("No transcripts available for this video")
-            
-            # First try to get transcript in target language
+
             try:
                 transcript = transcript_list.find_transcript([target_language])
                 return TextFormatter().format_transcript(transcript.fetch()), target_language
             except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)